scripts/build.sh

```diff
@@ -118,7 +119,8 @@ function GetOSVersion
 
     *)
       printf "[x] ${distribution_name} is not yet a supported distribution.\n"
-      return 1
+      # FIXME: remove workaround
```

Comment:
Maybe just add in a case for your operating system that makes `OS_VERSION` into `ubuntu1604` and print out a warning? Then this will be an officially supported workaround.

Reply:
I'm running Antergos Linux; weirdly enough, `cat /etc/issue` returns "Antergos Linux \r (\l)", which does weird things to any kind of string manipulation: `"${distribution_name}"` becomes "(\l)gos Linux". Maybe I can see about some other kind of check we can add.

Reply:
Perhaps `/etc/os-release/` would be a better file to check?